tests/ui/user/addUser.positive.test.ts

```diff
@@ -0,0 +1,60 @@
+import { test, expect } from '@playwright/test';
+import AddUserPage from '../../../pages/ui/addUserPage';
+import HomePage from '../../../pages/ui/homePage';
+import { faker } from '@faker-js/faker';
+import { Gender } from '../../../enums/gender.enum';
+import { UserApi } from '../../../services/api/user.api';
+import { UserDTO } from '../../../dto/userDTO';
+
+let addUserPage: AddUserPage;
+const userApi = new UserApi();
+let createdUsernames: string[] = [];
+
+const validUser: UserDTO = {
+  gender: Gender.Male,
+  name: faker.person.firstName('male'),
+  yearOfBirth: 1990
+};
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(`Forms/AddUser`);
+  addUserPage = new AddUserPage(page);
+});
+
+test.afterEach(async ({ request }) => {
+  if (!createdUsernames.length) return;
+
+  const response = await userApi.getUsers(request);
+  if (response.ok()) {
+    const users = await response.json();
+
+    for (const name of createdUsernames) {
+      const match = users.find((user: UserDTO) => user.name === name);
+      if (match) await userApi.deleteUser(request, match.id);
```

Comment:
it's better to always leave braces. In the future you or your colleague can add some code below and forget to add braces. And it's harder to read your code